lib/cli.js

```diff
@@ -41,6 +41,16 @@ var processArgs = function (argv, options, fs, path) {
     options.failOnEmptyTestSuite = options.failOnEmptyTestSuite === 'true'
   }
 
+  if (helper.isString(options.formatError)) {
+    var required = require(options.formatError)
```

Comment:
It would be good to wrap this into a `try catch` and provide a helpful error message if it fails.